core/src/main/java/org/apache/druid/java/util/metrics/JvmMonitor.java

```diff
@@ -108,10 +89,6 @@ private void emitThreadAllocationMetrics(ServiceEmitter emitter)
     }
   }
 
-  /**
-   * These metrics are going to be replaced by new jvm/gc/mem/* metrics
-   */
-  @Deprecated
```

Comment:
Don't know why it's marked as deprecated, and why you delete these

Reply:
This comment was deleted by mistake.